code/modules/keybindings/setup.dm

```diff
@@ -0,0 +1,55 @@
+/client
+	var/list/keys_held = list() // A list of any keys held currently
```

Comment:
Can you move this to `client_defines.dm`?
This is not the only instance in where variable definitions are spread around, but it's good to avoid it.